target/linux/ipq40xx/image/generic.mk

```diff
@@ -873,6 +873,18 @@ define Device/qxwlan_e2600ac-c2
 endef
 TARGET_DEVICES += qxwlan_e2600ac-c2
 
+define Device/tel_x1pro
+	$(call Device/FitImage)
+	DEVICE_VENDOR := Telco
+	DEVICE_MODEL := X1 Pro
+	SOC := qcom-ipq4019
+	KERNEL_SIZE := 4096k
+	IMAGE_SIZE := 31232k
+	IMAGE/sysupgrade.bin := append-kernel | append-rootfs | pad-rootfs | append-metadata
+	DEFAULT := n
```

Comment:
Why is this deactivated by default?

Reply:
Hmm, actually, I don't understand this `DEFAULT`; could you tell me what that is for please?

Reply:
The `DEFAULT := n` will deactivate this board by default in the build. We use it for example when the image does not fit onto the flash in the default configuration to not build the images with build bot.